### PR TITLE
feat: Add page layout components

### PR DESCRIPTION
**Pull request by rwharrisjr**
Adds page layout components:

- APageContainer
- APageHeader
- APageTitle
- APageDescription
- APageLabel
- APageTitleContent
- APageTitleContentLeft
- APageTitleContentRight
- APageTitleText

Additional components:

- ANetworkValue

### CODE REVIEW

File: docs/Playground.js [hunk 1]
```diff
@@ -41,7 +41,12 @@ const Playground = ({code, fullWidthPreview, noInline}) => {
             </div>
             <div
               className="pa-4 playground__preview"
-              style={{flexBasis: "50%", marginLeft: "50%", maxWidth: "50%"}}
+              style={{
+                flexBasis: "50%",
+                marginLeft: "50%",
+                maxWidth: "50%",
+                ...previewStyle
```
Comment (jonathanmcsweet) [inline docs/Playground.js:48]: Nice 😎

**Review by jonathanmcsweet — APPROVED** [review]
I read over all of the code and ran the dev environment to see these components in the Playground. I can't find any real issues but I did leave some comments about future features

**Review by wlorand3 — COMMENTED** [review]
These are looking great Robert — will issue some comments in the PR as well - would be great in a future iteration to somehow combine the existing "Layout" entries for ARow, ACol inside the new PageLayout or rename those for greater clarity;
It also might be worth calling out explicitly that APageTitle renders an H1 for its obvious SEO and page hierarchy benefits.

I think these will go a long way to ensuring consistency to the main app pages - personally, I'd love to see more APageDescriptions for all the XDR pages as single words like Deployments and Profiles have to carry all the weight to describe their functionality (but that's another topic) 

Re: consistency, Incident Details is quite the outlier with it's tags before the page title, breadcrumbs that are not breadcrumbs and the Mitre widget (screenshot), but this is obviously by design.

After taking a tour through most of the app pages, these new components look to be applicable to nearly all of them - 

Finally, did see a console warning about the data size of the next routes that you may already be aware of (screenshot)

Screenshot 2023-08-30 at 1 36 08 PM
Screenshot 2023-08-30 at 1 43 54 PM

File: framework/components/ANetworkValue/ANetworkValue.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ANetworkValue.scss";
+
+const baseClass = "a-network-value";
+
+const ANetworkValue = ({className: propsClassName, children, ...rest}) => {
```
[the remark this answers is not in the document]
Comment (rwharrisjr) [reply framework/components/ANetworkValue/ANetworkValue.js:7]: Opened https://github.com/cisco-sbg-ui/magna-react/issues/508 for future enhancements. Magnetic doesn't seem to specify much about it other than the font, so seems like fair play to add to it at some point.

**Comment by rwharrisjr** [comment]
I'll work on some examples for adding in ARow/ACol. I'm not sure about combining them with the page layout components, but perhaps renaming Page Layout to Page Container might help.

Good call on the incidents page, I'll add a "pre title" of some sort 

I'm not quite sue how to resolve the warning about the routes.. I've just been ignoring that since it doesn't affect anything

**Comment by androche** [comment]
Looks great in examples - the only thing I can think of is maybe adding ellipses for titles instead of wrap and handling the container width, or maybe just a prop options for it.
Screenshot 2023-09-01 at 1 12 47 PM

Screenshot 2023-09-01 at 1 11 51 PM

**Review by androche — APPROVED** [review]

**Pull request merged**